### PR TITLE
Update Czech based on other projects style guides

### PR DESCRIPTION
**Pull request by MikkCZ**
**me (@MikkCZ) squash this before merge!**

In this phase I want to update the style guide based on the [manual on wiki.l10n.cz](http://wiki.l10n.cz/N%C3%A1vody_pro_p%C5%99ekladatele) (excluding the referenced articles) and style guides available for other projects (WordPress, Facebook and Microsoft) we have listed [here](http://wiki.l10n.cz/Pom%C5%AFcky_pro_p%C5%99ekladatele#Stylistick.C3.A9_p.C5.99.C3.ADru.C4.8Dky_ciz.C3.ADch_projekt.C5.AF).

- [x] info from the [manual on wiki.l10n.cz](http://wiki.l10n.cz/N%C3%A1vody_pro_p%C5%99ekladatele)
- [x] important points from the [Drupal style guide](http://preklady.drupal.cz/pravidla)
- [x] ~important points from the [WordPress style guide](https://docs.google.com/document/d/1bVTr-Qw2jiXr6_vzztl4SoZTi0InHjkMGkLa7UEbu3s/edit)~ (nothing more to take)
- [x] important points from the [Facebook style guide](https://www.facebook.com/translations/style_guides)
- [x] important points from the [Microsoft style guide](https://www.microsoft.com/en-us/language/StyleGuides)

fix https://github.com/MozillaCZ/ukoly/issues/14

### CODE REVIEW

**Comment by MikkCZ** [comment]
I have fixed the typo and my English mistakes. @FHRNet please re-review.

File: cs/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Toto je úvodní stránka překladatelské příručky české komunity Mozilly.
 
 ### Kontrola pravopisu
 
-Kontrola pravopisu není zbytečný luxus. Odstraní řadu překlepů. Ve Firefoxu můžete použít [doplněk pro kontrolu překlepů](https://support.mozilla.org/cs/kb/jak-pouzit-kontrolu-pravopisu) a pomůže vám i [služba Korektor](https://support.mozilla.org/cs/kb/jak-pouzit-kontrolu-pravopisu#w_online-sluaaba-a-doplnluk-korektor). Hlavně u delších textu se nebojte poprosit někoho dalšího, aby po vás překlad přečetl, nebo si jej přečtěte znovu sami druhý ten.
+Kontrola pravopisu není zbytečný luxus. Odstraní řadu překlepů. Ve Firefoxu můžete použít [doplněk pro kontrolu překlepů](https://support.mozilla.org/cs/kb/jak-pouzit-kontrolu-pravopisu) a pomůže vám i [služba Korektor](https://support.mozilla.org/cs/kb/jak-pouzit-kontrolu-pravopisu#w_online-sluaaba-a-doplnluk-korektor). Hlavně u delších textů se nebojte poprosit někoho dalšího, aby po vás překlad přečetl, nebo si jej přečtěte znovu sami druhý ten.
```
Comment (FHRNet) [inline cs/README.md]: Oops, one more typo: `ten` => `den`

Comment (MikkCZ) [reply cs/README.md]: One more commit. ;)

**Comment by MekliCZ** [comment]
r+

**Pull request merged**